common/src/main/scala/explore/model/AppConfig.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import sttp.model.Uri
+import io.circe._
+import io.circe.generic.semiauto._
+import explore.model.encoders._
+import explore.model.decoders._
+
+case class AppConfig(exploreDBURI: Uri, odbURI: Uri)
```

Comment:
Maybe this should have something like `isStaging` to indicate in the UI

Reply:
Good idea. Or an `Environment` `enum` ?